packages/components/Menu/src/MenuList/MenuList.tsx

```diff
@@ -58,6 +58,19 @@ export const MenuList = compose<MenuListType>({
     }, []);
 
     return (_final: MenuListProps, children: React.ReactNode) => {
+      const childrenWithSet = React.Children.map(children, (child, index: number) => {
+        if (React.isValidElement(child)) {
+          return React.cloneElement(child, {
+            accessibilityPositionInSet: index, // win32
+            accessibilityPosInSet: index, // windows
+            accessibilitySetSize: React.Children.toArray(children).length, //win32
+            accessibilitySizeOfSet: React.Children.toArray(children).length, //windows
```

Comment:
Would this be better as a Platform.Select()?

Reply:
Platform.select doesn't work for win32. I can do a conditional but unless the extra props are a problem, it seems fine to pass everything in

Reply:
I don't think it's a problem, though it'll be something to keep in mind for RN upgrades (in case win32 implements the windows props in the future, or macOS does, etc..). I'll consider it not blocking I guess.

Side note, do you know why these differ between the windows platforms?

Reply:
Yeah, in the future I expect the win32 props to align with windows.

I don't know why they don't align, my guess would be that they were both created around the same time and didn't consult each other.